torch/csrc/jit/ir.cpp

```diff
@@ -97,6 +77,51 @@ std::string PythonOp::name() const {
   return getPythonName(pyobj.get(),is_legacy);
 }
 
+}}
+
+#else
+
+namespace torch { namespace jit {
+
+std::ostream& printPyObject(std::ostream & out, const THPObjectPtr& obj) {
+  throw std::runtime_error("Trying to print Python object from a C++ build");
+}
+
+std::string PythonOp::name() const {
+  throw std::runtime_error("Trying to call PythonOp::name from a C++ build");
+  return std::string();
+}
+
+}}
```

Comment:
Please add comments which say what namespaces are you closing (both here and above)

Reply:
Done